packages/grafana-ui/src/themes/GlobalStyles/utilityClasses.ts

```diff
@@ -33,6 +34,26 @@ function buttonSizeMixin(paddingY: string, paddingX: string, fontSize: string, b
   };
 }
 
+function widthMixin(theme: GrafanaTheme2, max: number) {
+  let result: CSSInterpolation = {};
+  for (let i = 1; i <= max; i++) {
+    result[`.width-${i}`] = {
+      width: `${theme.spacing(2 * i)} !important`,
```

Comment:
Nit: The `${theme.spacing(2 * i)} !important` could be saved into a var and calculated only once per loop iteration.